internal/operators/mce/config.go

```diff
@@ -1,8 +1,66 @@
 package mce
 
+import (
+	"embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/openshift/assisted-service/internal/common"
+)
+
+//go:embed ocp_mce_release_matrix.json
+var jsonFile embed.FS
+
+func getMCEVersion(openshiftVersion string) (*string, error) {
+
+	type OCPMCEReleaseMatrix struct {
+		OpenshiftVersion string   `json:"openshift_version"`
+		MceVersions      []string `json:"mce_versions"`
+	}
+
+	baseVersion, err := common.GetBaseVersion(openshiftVersion)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get base version of %s", openshiftVersion)
+	}
+
+	content, err := jsonFile.ReadFile("ocp_mce_release_matrix.json")
```

Comment:
where this file is from ?

maybe we should put it in a const for  future name chagnes ?

Reply:
Will do thanks